gnoi/os/client_test.go

```diff
@@ -223,7 +273,7 @@ func TestInstall(t *testing.T) {
 				}},
 			}
 			fileReader = test.reader
-			if err := client.Install(context.Background(), "", "version", 100*time.Millisecond); fmt.Sprintf("%v", err) != fmt.Sprintf("%v", test.err) {
+			if err := client.Install(context.Background(), "", "version", test.timeout); fmt.Sprintf("%v", err) != fmt.Sprintf("%v", test.err) {
```

Comment:
have you tried with: err.String() != test.err.String()

Reply:
It was just more concise to do it this way since it also includes nil checking. I can split it up if you want.

Reply:
It's personal preference hence please feel free to leave it as is.